setup.py

```diff
@@ -55,9 +55,12 @@
         "dev": [
             "dm-haiku",
             "flax",
+            # TODO: bump funsor version before the release
             "funsor @ git+https://github.com/pyro-ppl/funsor.git@d5574988665dd822ec64e41f2b54b9dc929959dc",
             "graphviz",
-            "tensorflow_probability",
+            # TODO: change this to tensorflow_probability>0.12.1 when the next version
+            # of tfp is released. The current release is not compatible with jax>=0.2.12.
+            "tfp-nightly",
```

Comment:
This is due to tensorflow_probability==0.12.1 still uses `jax.custom_transforms`, which has been deprecated for a while and has been removed in the last JAX release